v2/AdminCore.Services/EventService.cs

```diff
@@ -378,5 +364,17 @@ private Event BuildNewEvent(int employeeId)
       };
       return newEvent;
     }
+
+    private EventMessage BuildRejectEventMessage(string message)
```

Comment:
Seems like this could be reused for different event message types, could pass it as a parameter?